src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/legaladvisor/OrderCreator.java

```diff
@@ -90,6 +91,7 @@ public CallbackResponse prepopulateOrder(CallbackParams callbackParams) {
         data.put(DOC_UPLOAD_FOR_PARTY, BOTH.name());
         data.put(EYEWITNESS_UPLOAD_FOR_PARTY, BOTH.name());
         data.put(PAPER_DETERMINATION, NO.name());
+        data.put(GRANT_EXPERT_REPORT_PERMISSION, NO);
```

Comment:
Not `NO.name()`?

Reply:
It's an enum being cast to a string so name() is not needed. Was in 2 minds about cleaning all the other calls up